javascript.ru.md

```diff
@@ -58,15 +58,15 @@ JavaScript CodeStyle
   * В именовании аббревиатур соблюдаем `CamelCase`. Например: `Json`, `Xml`.
 
 ##<a name="3"></a>Объявление переменных
-  * Все переменные объявляются с `var`.
+  * Все переменные должны быть объявлены перед использованием.
+  * Следует избегать как неявного использования глобальных переменных, так и глобальных переменных в принципе.
```

Comment:
I think, you must remove this line, it doesn't conform English version.
And overlaps with other rules.